community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -94,6 +94,9 @@
             "dbms.cypher.planner",
             options( "COST", "RULE", DEFAULT ), DEFAULT );
 
+    @Description( "Set this to specify the behavior for conformance to planner hints." )
```

Comment:
I would remove the `planner` from description, since the property name doesn't suggest coupling with planning.